vdb-bench-assembly/plugins/vdb-bench-widgets/gitCredentialsControl.js

```diff
@@ -70,6 +88,34 @@
             vm.setSelected(vm.repositories[0]);
         }
 
+        vm.httpParamsComplete = function() {
+            if ( _.isEmpty( vm.repo ) || _.isEmpty( vm.repo.parameters ) ) {
+                return false;
+            }
+
+            return !_.isEmpty( vm.repo.parameters[ 'repo-password-property' ] ) && !_.isEmpty( vm.repo.parameters[ 'repo-username-property' ] );
+        };
+
+        vm.sshParamsComplete = function() {
+            return vm.sshPrivateKeyParamsComplete() || vm.sshPasswordParamsComplete();
+        };
+
+        vm.sshPasswordParamsComplete = function() {
+            if ( _.isEmpty( vm.repo ) || _.isEmpty( vm.repo.parameters ) ) {
+                return false;
+            }
+
+            return !_.isEmpty( vm.repo.parameters[ 'repo-password-property' ] ) && !_.isEmpty( vm.repo.parameters[ 'repo-known-hosts-property' ] );
```

Comment:
Again questionable whether known-hosts is a required pre-requisite for ssh. Its required on my ssh configuration but no guarantee its essential for all setups.